font-codegen/src/parsing.rs

```diff
@@ -365,6 +366,18 @@ pub(crate) struct BitFlags {
     pub(crate) variants: Vec<RawVariant>,
 }
 
+#[derive(Debug, Clone)]
+pub(crate) enum ExternTarget {
```

Comment:
maybe `ExternType`, to match `FieldType`? I also might move the ident out of here and into `Extern` directly, since every possible extern item will have a name.

Reply:
Will do, good catch